src/client/src/orangehrmAttendancePlugin/pages/ViewEmployeeAttendanceSummary.vue

```diff
@@ -133,7 +133,7 @@ export default {
       return data.map(item => {
         return {
           id: item.empNumber,
-          empName: $tEmpName(this.employee, {
+          empName: $tEmpName(item, {
```

Comment:
Have you done developer testing on this? confirmed working?

Reply:
Working